tests/unit/components/sl-tab-pane-test.js

```diff
@@ -1,78 +1,90 @@
 import Ember from 'ember';
 import { moduleForComponent, test } from 'ember-qunit';
 import startApp from '../../helpers/start-app';
-import { contains } from '../../helpers/sl/synchronous';
 
 var App;
 
-moduleForComponent( 'sl-tab-pane', 'Unit - component: sl-tab-pane', {
+moduleForComponent( 'sl-tab-pane', 'Unit | Component | sl tab pane', {
     needs: [ 'component:sl-tab-panel' ],
 
-    beforeEach: function() {
+    beforeEach() {
         App = startApp();
     },
 
-    afterEach: function() {
+    afterEach() {
         Ember.run( App, App.destroy );
-    }
+    },
+
+    unit: true
 });
 
 test( 'Expected default classes are applied', function( assert ) {
-    var $component = this.render();
-
-    assert.ok( contains( $component.prop( 'class' ), [ 'sl-tab-pane', 'tab-pane' ] ), 'Default classes are not correctly applied' );
```

Comment:
I believe this is the last use of the contains() helper and of any of the sl-ember-test-helpers.  If so, remove the sl-ember-test-helpers dependency and registration of it in the test/app file